airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -2916,6 +3149,331 @@ schemas:
           - "null"
           - string
 
+  engagements_calls:
+    $schema: "http://json-schema.org/draft-07/schema#"
+    type:
+      - "null"
+      - object
+    properties:
+      id:
+        description: Unique identifier for the call engagement.
+        type:
+          - "null"
+          - string
+      createdAt:
+        description: Date and time when the call engagement was created.
+        type:
+          - "null"
+          - string
+        format: date-time
```

Comment:
not sure if this needed for the streams you are migrating, but previously we were adding `__ab_apply_cast_datetime: false` to align with original format as it was in python implementation for `crealedAt`, `updatedAt` etc.

Reply:
ah yes good catch i'll add that like the other streams